docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,44 @@
+# AntennaAttributionDocumentReporter
+The AntennaAttributionDocumentReporter generates an attribution document in .pdf format by leveraging the [Eclipse 
+Antenna][1] project. It generates a blank PDF file without any configuration, but the attribution document can be 
+customized by providing a template JAR, or a directory with template PDF files. 
```

Comment:
Maybe "template bundle JAR"? I'm not sure if it's common to use "bundle" here, but I tend to like it as it emphasizes that is bundles the otherwise separate PDFs. If using "template bundle JAR" here, we should also use that term consistently throughout the document.

Reply:
Done.